libstratum/EthStratumClient.cpp

```diff
@@ -532,14 +527,14 @@ bool EthStratumClient::submit(Solution solution) {
 		boost::asio::placeholders::error));
 	if (m_stale)
 	{
-		cwarn << "Stale solution found. Submitted to" << p_active->host;
+		cwarn << EthYellow "Stale solution submitted to" + p_active->host + EthReset;
```

Comment:
To be consistent we should always output the complete identifier of the remote host endpoint thus indicating :

`cwarn << EthYellow "Stale solution submitted to" + p_active->host + ":" + p_active->port + EthReset;`

This makes sense as for example ethermine.org offers multiple ports on the same host name. (eu1.ethermine.org:4444 and eu1.ethermine.org:14444)